Library/Tales of the Solar Patrol/Solar Patrol Disadvantages.adq

```diff
@@ -0,0 +1,18 @@
+{
+	"type": "advantage_list",
+	"version": 2,
+	"id": "bb06d599-cede-4e3f-b08f-add7f5a25226",
+	"rows": [
+		{
+			"type": "advantage",
+			"id": "a0012a07-0880-4fea-acea-71f54748035c",
+			"name": "Last Stand",
+			"physical": true,
+			"base_points": -10,
+			"calc": {
+				"points": -10
+			},
+			"notes": "You are more than just willing to die for the cause – you yearn for the chance, though you’ll never admit it publicly (lest the psych boys find you and deny you your opportunity for glory!). \n\nAny time you have a chance to sacrifice yourself nobly in the pursuit of a mission – to save a comrade or to accomplish an order of the Patrol – you must make a self-control roll or choose to give your all for the greater good. This only applies in situations when your death would offer at least some advantage over other means of solving the problem (though it need not be the only means!) or would otherwise look “heroic,” as opposed to “stupid.” (No one wants to be remembered in the history books as an example of what not to do.) \n\nFurthermore, you must act so as to not take others with you. If you think piloting your ship directly into the maw of the Overlord’s Neutronite Cannon is the best way to save the Fleet, you do so only after conking your co-pilot on the head and tossing him into a lifeboat."
```

Comment:
Please either eliminate this note or pare it down to the essentials. While having the full text from the book (I assume that's what this is -- I don't have this particular supplement) is fine for your own copy of the data, the version that makes its way into the master library needs to be more conservative to keep Steve Jackson Games happy.

Reply:
Thanks Richard, I've edited the text to a brief description and a page reference.

Reply:
Rather than putting the page reference in the notes, how about adding it to the Advantage entry instead?

Reply:
I've had a go. I'm finding Github a bit confusing, so I don't know if the note with the page reference has been removed or not.

Reply:
Looks like you’ve removed the note entirely and added the page reference. Assuming you didn’t want a reduced note at all, the changes you’ve made look good. I still need to take a quick look through the various new sheets and templates, but will do that a bit later today. Assuming I don’t find any other issues, I’ll merge these changes. Thanks. Oh, if you decide you want to re-add a subset of the original note, let me know and I’ll hold off on merging until you do.